src/components/modals/TransferCardModal.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <transition name="modal-fade">
+    <div class="modal__backdrop">
+      <div
```

Comment:
the ccbutton class should be used so that the buttons become our parallelogram shaped buttons (this problem is also at the profile page)

Reply:
![image](https://user-images.githubusercontent.com/64218491/170113962-d4fb001a-c09c-4957-9176-c6cfc9a17c3c.png)
That's just a problem in the account page.